vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/MapBase.java

```diff
@@ -141,6 +142,11 @@ public Registration addViewMoveEndEventListener(
         return addListener(MapViewMoveEndEvent.class, listener);
     }
 
+    public Registration addMapClickEventListener(
```

Comment:
Missing Javadoc

Also let's remove `Map` from the name, to keep this consistent with `addViewMoveEndEventListener`